src/main/java/io/gravitee/repository/jdbc/management/JdbcCommandRepository.java

```diff
@@ -159,16 +157,16 @@ public Command update(Command item) throws TechnicalException {
     @Override
     public List<Command> search(CommandCriteria criteria) {
         LOGGER.debug("JdbcCommandRepository.search({})", criteria);
-        JdbcHelper.CollatingRowMapper<Command> rowMapper = new JdbcHelper.CollatingRowMapper<>(ORM.getRowMapper(), CHILD_ADDER, "id");
+        JdbcHelper.CollatingRowMapper<Command> rowMapper = new JdbcHelper.CollatingRowMapper<>(getOrm().getRowMapper(), CHILD_ADDER, "id");
         final StringBuilder query = new StringBuilder(
-                "select * from " + escapeReservedWord("commands") + " c " +
-                "left join command_acknowledgments ca on c.id = ca.command_id " +
-                "left join command_tags ct on c.id = ct.command_id " +
+                "select * from " + escapeReservedWord(this.tableName) + " c " +
```

Comment:
escape the table name in the constructor
```suggestion
                getOrm().getSelectAll() + " c " +
```